RELEASING.md

```diff
@@ -0,0 +1,144 @@
+        description: Release type
+        required: true
+        default: "auto"
+        type: choice
+        options:
+        - minor
+        - patch
+      release_branch:
+        description: Branch to release
+        required: true
+        default: master
+        type: string
+      quay_repository:
+        description: Quay repository
+        type: string
+        default: quay.io/kiali/kiali
+        required: true
+
+jobs:
+  initialize:
+    name: Initialize
+    runs-on: ubuntu-20.04
+    outputs:
+      target_branch: ${{ github.ref_name }}
+      release_type: ${{ steps.release_type.outputs.release_type }}
+      release_version: ${{ steps.release_version.outputs.release_version }}
+      branch_version: ${{ steps.branch_version.outputs.branch_version }}
+      next_version: ${{ steps.next_version.outputs.next_version }}
+      quay_tag: ${{ steps.quay_tag.outputs.quay_tag }}
+    steps:    
+    
+    ...
+    
+    - name: Log information
+      run: |
+        echo "Release type: ${{ steps.release_type.outputs.release_type }}"
+
+        echo "Release version: ${{ steps.release_version.outputs.release_version }}"
+
+        echo "Next version: ${{ steps.next_version.outputs.next_version }}"
+
+        echo "Branch version: ${{ steps.branch_version.outputs.branch_version }}"
+
+        echo "Quay tag": ${{ steps.quay_tag.outputs.quay_tag }}
+
+  ...
+
+  release:
+    name: Release
+    if: ${{ needs.initialize.outputs.release_type != 'skip' && ((github.event_name == 'schedule' && github.repository == 'kiali/kiali') || github.event_name != 'schedule') }}
+    runs-on: ubuntu-20.04
+    needs: [initialize, build_backend, build_frontend, integration_tests_backend, integration_tests_frontend]
+    env:  
+      RELEASE_VERSION: ${{ needs.initialize.outputs.release_version }}
+      BRANCH_VERSION: ${{ needs.initialize.outputs.branch_version }}
+      NEXT_VERSION: ${{ needs.initialize.outputs.next_version }}
+      RELEASE_BRANCH: ${{ github.event.inputs.release_branch || github.ref_name }} 
+      QUAY_TAG: ${{ needs.initialize.outputs.quay_tag }}
+    steps:
+    
+    ...
+
+```
+
+By design, the release pipelines can be triggered manually and also are scheduled to trigger on every Friday.
+
+When triggering manually, some parameters will be required, these are the inputs to the pipeline and every pipeline declare defaults values.
+
+When the pipeline is triggered automatically, it will detect if it needs to publish the release or not.
+
+Note: In the past, we run the release pipeline every Friday using a cron, then the snapshots or the minor release were created. With this pipelines is the same, we need to run the job every Friday (cron does not support running for example every 3 weeks), but as we removed snapshots, there is not much to do on the first two weeks of our sprint, only to skip the release. On the third week of the sprint, he cron will fire on the Friday, and it will create a minor release.
```

Comment:
```suggestion
Note: In the past, we ran the release pipeline every Friday using a cron job, then the snapshots or the minor release were created. It is the same with these new pipelines - we need to run the job every Friday (the GitHub Actions cron does not support running, for example, every 3 weeks). But we have now removed the building/releasing of weekly snapshots. Because weekly snapshots are no longer released, there is not much to do the first two weeks of our sprint (the pipeline will simply skip doing any release those first two weeks). On the third week of the sprint, the cron will fire on Friday, and at that time it will create a minor release.
```